### PR TITLE
Tweak issue forms

### PR DESCRIPTION
**Pull request by chadwhitacre**
1. Expand "Bug Report" to "Problem Report" to also cover support requests. With #1151 we're explicitly asking for support requests to be filed here in Issues.
1. Rename files to keep up with (1) and for clarity.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/problem-report.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ body:
   - type: markdown
     attributes:
       value: |-
-        ## Thanks 🙏
+        ## Thanks
```
Comment (chadwhitacre) [inline .github/ISSUE_TEMPLATE/problem-report.yml:43]: Sentry style guide sez no emoji in corporate voice.

**Comment by chadwhitacre** [comment]
You probably can't review PRs, can you @aminvakil?

**Review by aminvakil — APPROVED** [review]
The PR looks good to me!
I don't think my approve counts if that's what you mean, but let's see :)

Well it didn't :(

**Comment by chadwhitacre** [comment]
Overrode review requirement for now. Thanks @aminvakil. :)

**Pull request merged**